src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1224,6 +1235,20 @@ static bool processSubscriptions
       BSONObj update = BSON("$set" << BSON(CSUB_LASTNOTIFICATION << getCurrentTime()) <<
                             "$inc" << BSON(CSUB_COUNT << 1));
 
+
+      //
+      // Saving lastNotificationTime for cached subscription
```

Comment:
Probably it doesn't matter, but from a logic program point of view, the lastNotification time should be modified in the cache *after* the update has been done in the DB.